docs/standard/security/fips-compliance.md

```diff
@@ -0,0 +1,29 @@
+---
+title: .NET Core FIPS compliance
+description: Explains .NET Core FIPS compliance
+ms.date: 11/20/2019
+author: "Rick-Anderson"
+ms.author: "riande"
+---
+
+# .NET Core FIPS compliance
+
+.NET Core:
+
+* Does **not** implement any cryptographic primitives.
+* Passes cryptographic primitives calls through to the standard modules the underlying operating system provides.
+* Does **not** enforce FIPS compliance in .NET Core apps.
+* Does **not** stop developers from using non-FIPS compliant algorithms, even on systems configured for FIPS compliance.
+
+It's the responsibility of the system administrator to configure FIPS compliance for an operating system.
+
+If code is written for a FIPS-compliant environment, it's the responsibility of the developer to esure that:
+
+* Non-compliant FIPS algorithms are not used.
```

Comment:
seems over the top to have a one bullet list

Reply:
Rewrite so it's not a long sentence. One bullet better than long sentence. Maybe @blowdart  or  @bartonjs  can suggest another bullet.

Reply:
that's why I offered a rewrite to make it smaller.

Reply:
"FIPS compliant" is a very difficult phrase.  In order to be "FIPS Compliant" you have to be using "FIPS Approved" algorithms from a "FIPS Certified" implementation (which frequently requires that implementation be running in a "FIPS Mode") -- but it only applies during a "FIPS Sensitive" (my phrase, not real) context.

e.g. FIPS, AFAIU, doesn't care if you use MD5 to set the Content-MD5 HTTP header (when the alternative is no data checksum).  But you can't do an RSA+MD5 signature, because MD5 can't be used in a cryptographic context.

Reply:
I changed the title to 
**Explains .NET Core Federal Information Processing Standard (FIPS) compliance**

FIPS folks won't have a problem. This article isn't aimed at new dev's looking to try something new.